src-tauri/src/hardware/hardware_status_monitor.rs

```diff
@@ -357,6 +363,57 @@ impl HardwareStatusMonitor {
         Ok(platform_devices)
     }
 
+    // This method will decide if gpu mining is recommended by which we mean if it should be enabled by default in config mining
+    // It will have two steps verification:
+    // Checking raw_detected_devices in graxil which includes device_type ( INTEGRATED or DEDICATED )
+    // Checking system memory with sysinfo
+    // If there is at least one dedicated gpu and system memory is above 16GB we recommend enabling gpu mining
+    pub async fn decide_if_gpu_mining_is_recommended(&self) -> Result<(), Error> {
+        let mut is_dedicated_gpu_found = false;
+        let mut is_system_memory_above_16gb = false;
```

Comment:
Should be >=16gb

16GB and integrated is a valid state to leave GPU enabled.

Which I can see on line 389, it is a `>=` check, but the name of this variable makes it slightly misleading.